packages/components/src/components/telekom/app-header/app-header.tsx

```diff
@@ -218,11 +218,8 @@ export class Header {
       readData(this.mainNavigation),
       this.activeRouteId
     );
-    const isActive = (item) =>
-      rootNode &&
-      rootNode.id === item.id &&
-      !this.visibleMegaMenu &&
-      this.visibleMegaMenu !== null;
+    const isActive = (item) => rootNode && rootNode.id === item.id;
+    console.log(rootNode, this.visibleMegaMenu);
```

Comment:
🎯

Reply:
sry i will remove it

Reply:
this PR includes 2 fixes, one from Maomao and one from me , this code is part of @maomaoZH change :)

Reply:
done and pushed